core/web/loader/getters.go

```diff
@@ -37,6 +38,24 @@ func GetChainByID(ctx context.Context, id string) (*commonTypes.ChainStatusWithI
 	return &chain, nil
 }
 
+// GetChainByRelayID fetches the chain by it's relayId.
+func GetChainByRelayID(ctx context.Context, id string) (*commonTypes.ChainStatusWithID, error) {
+	ldr := For(ctx)
+
+	thunk := ldr.ChainsByRelayIDLoader.Load(ctx, dataloader.StringKey(id))
```

Comment:
what does `thunk` mean as a var?

Reply:
Great question, in this context, thunk is just promise, but mainly i was [following the rest](https://github.com/smartcontractkit/chainlink/blob/dc96cdcdd0c19726fe86702f8eae429e04cfa89f/core/web/loader/getters.go#L81-L81) of the data loaders, they seem to call the value returned by `load` thunk